### DISCUSSION TITLE
I cant get it to work

### DISCUSSION THREAD

**Discussion by MinisterK** [onimp/oni_multiplayer#8] [Q&A] [answered]
when I installed the mod it says no compatible mod found

**Comment by zuev93** [comment]
Hi MinisterK,

FYI mod is still under active development and not fully functional yet.

However I'll be happy to help you out. Could you help me and answer on following questions? 

- What version of ONI do you have?
- Do you have DLC installed?
- What other mods do you have installed?
- Could you provide a screenshot?

Best regards, zuev93.

**Reply by MinisterK** [reply]
Hi zuev93 I understand that the mod is still work in progress and here are my answers

I have the latest version of oxygen not included o steam installed
I have no dlc
My other mods are uninstalled
![20230221180521_1](https://user-images.githubusercontent.com/125838905/220503004-052732ca-fc35-407c-b380-ffe8822cf6a2.jpg)

**Comment by zuev93** [comment, marked as answer]
I'm slightly confused to see mod name as `oni_multplayer-master` did you use release zip archive? Or download entire repo?

To install mod you need use this link https://github.com/zuev93/oni_multiplayer/releases/tag/v0.0.3 and choose file called `MultiplayerMod.zip` [MultiplayerMod.zip](https://github.com/zuev93/oni_multiplayer/releases/download/v0.0.3/MultiplayerMod.zip)
